local_archives/nwp/cams/cams_global.py

```diff
@@ -0,0 +1,264 @@
+from cdsapi import Client
+
+from local_archives.partitions import InitTimePartitionsDefinition
```

Comment:
For this, is it easy to run backfills with specific variables in the UI or somewhere? As our current backfill is only pulling 5 variables, and that had to be a code change.

Reply:
The way its defined at the moment it will pull all the variables from the get go - is it more desirable to have the variables configurable?